tst/run.sh

```diff
@@ -0,0 +1 @@
+vendor/bin/phpunit --testdox
```

Comment:
We already have `php artisan test`, which does toggle testdox

Reply:
Okay, I can remove this.